widget/popup.go

```diff
@@ -90,10 +90,9 @@ func (p *PopUp) CreateRenderer() fyne.WidgetRenderer {
 		return &modalPopUpRenderer{popUp: p, bg: bg, objects: []fyne.CanvasObject{bg, p.Content}}
 	}
 
-	shadow := newShadow(shadowAround, theme.Padding()*2)
 	bg := canvas.NewRectangle(theme.BackgroundColor())
-	objects := []fyne.CanvasObject{shadow, bg, p.Content}
-	return &popUpRenderer{popUp: p, shadow: shadow, bg: bg, objects: objects}
+	objects := []fyne.CanvasObject{bg, p.Content}
+	return &popUpRenderer{baseRenderer: newBaseRenderer(objects, 4), popUp: p, bg: bg}
```

Comment:
Should this '4' be a constant, or theme.Padding()?

Reply:
As the level is a kind of elevation it should not depend on the theme.

Reply:
I agree that it should not be defined by theme. But I also think a const makes sense to avoid magic numbers. Something like "elevationButton" or similar? Maybe we should reference these elevations? https://storage.googleapis.com/spec-host/mio-staging%2Fmio-design%2F1584058305895%2Fassets%2F0B6xUSjjSulxceF9udnA4Sk5tdU0%2Fbaselineelevation-chart.png